LEAF_Nexus/admin/index.php

```diff
@@ -15,8 +15,7 @@
 */
 error_reporting(E_ERROR);
 
-require_once '../globals.php';
-require_once LIB_PATH . '/loaders/Leaf_autoloader.php';
+require_once '/var/www/html/app/libs/loaders/Leaf_autoloader.php';
```

Comment:
Are absolute paths necessary? This can cause issues if the environment changes in the future.

Reply:
It is for the meantime. Until we can move these files to  the app folder and make them all single source of truth there is no way to know the total number of folders deep we are. When they move we can then change these back to relative paths.

We're trying to do this move in small incremental steps to make it less overwhelming for the review process. Plus it makes testing easier. All these changes will be full regression testing but if we run into something it will be easier to debug if we move things over slowly. We are also moving things in such a way as to make sure everything is backward compatible.

Reply:
The part I'm concerned with is:
```
/var/www/html
```

Would it be feasible to do something like:
```
require_once HTML_ROOT . '/app/libs/loaders/Leaf_autoloader.php';
```